install_config/aggregate_logging_sizing.adoc

```diff
@@ -205,10 +205,25 @@ Administrative Elasticsearch Operations] section for more in-depth information.
 
 include::install_config/aggregate_logging.adoc[tag=elasticsearch-ha]
 
+[[master-events-aggregated-to-efk-as-logs]]
+=== Master Events are Aggregated to EFK as Logs
+
+The *eventrouter* pod scrapes the events from kubernetes API and and outputs to
+*STDOUT*. The *fluentd* plug-in transforms the log message and sends it to
+Elasticsearch (ES).
+
+Enable `openshift_logging_install_eventrouter` by setting it to `true`. It is
+off by default. *Eventrouter* is deployed to the default namespace. Collected
+information is in operation indices of ES and only cluster administrators have
+visual access.
+
+See the link:https://github.com/openshift/origin-aggregated-logging/blob/master/docs/proposals/kube_events_design_doc.md[design
```

Comment:
We avoid linking to GitHub repos per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites; these links need special approval from the Support team.